### PR TITLE
#28: Enhance command history in CLI

### PR DESCRIPTION
**Pull request by prathameshkoshti**
Closes https://github.com/DiceDB/playground-web/issues/28

### Changes made in this PR:
- Removed duplicate event key pressed check from the function `handleCommandWrapper`
- Navigate through the command history based on current user input

https://github.com/user-attachments/assets/76ba93ef-acff-4131-9d42-b2bf3403876c

Test case coverage:
Screenshot 2024-10-04 at 12 41 29 PM

### CODE REVIEW

**Comment by prathameshkoshti** [comment]
Hey @lucifercr07 👋 , can you please review this PR.

File: src/components/CLI/hooks/useCli.tsx
Comment (lucifercr07) [inline src/components/CLI/hooks/useCli.tsx:1]: @prathameshkoshti thanks a lot for your contributions, can you please add some tests around these?

Comment (prathameshkoshti) [reply src/components/CLI/hooks/useCli.tsx:1]: @lucifercr07, by adding tests do you mean writing test cases using testing library something like jest?

Comment (lucifercr07) [reply src/components/CLI/hooks/useCli.tsx:1]: Yes, we can use `jest` library.

Comment (prathameshkoshti) [reply src/components/CLI/hooks/useCli.tsx:1]: @lucifercr07 should I create a new PR for jest setup or use this same PR

Comment (lucifercr07) [reply src/components/CLI/hooks/useCli.tsx:1]: You can create a new PR.

Comment (lucifercr07) [reply src/components/CLI/hooks/useCli.tsx:1]: @prathameshkoshti please add jest tests for these changes.

Comment (prathameshkoshti) [reply src/components/CLI/hooks/useCli.tsx:1]: Sure 👍

**Review by lucifercr07 — APPROVED** [review]

**Pull request merged**